www/src/components/navigation-mobile.js

```diff
@@ -51,7 +51,7 @@ const MobileNavigation = () => (
       bottom: 0,
       display: `flex`,
       fontFamily: `header`,
-      height: `headerHeight`,
+      height: `headerHeight + env(safe-area-inset-bottom)`,
```

Comment:
Awww thanks for taking care of this @lekterable! 🤗 🙏 
As I'm about to give this a shot on localhost — we don't need to `calc` here, and/or a `@supports` media query?

Reply:
Ah yeah, _looks_ like this fixes things because with this change, we now supply an invalid `height` — not setting it at all anymore:

![image](https://user-images.githubusercontent.com/21834/67208354-aa4a5700-f415-11e9-8fb5-98125ed99e27.png)

This is fine though — we are explicitly setting the height of the navigation items/hrefs to `headerHeight` anyway 🤦‍♂ 😅 

![image](https://user-images.githubusercontent.com/21834/67208380-b7674600-f415-11e9-9277-aa26bc99102a.png)

Reply:
So, I'm assuming we can just remove the whole `height` declaration and things should work fine for iPhone X  — may I ask you to test that @lekterable, and update the PR if this works? Thank you! 🙏

Reply:
OMG 🤦‍♂ I knew it, that was my first thought! I was like "Oh yeah it's really weird that it works without any calc needed, but at the same, it's super cool, maybe they are using some super cool css-in-js library for it or something, so let's just go with it"

Reply:
I'll test with height removed right now!

Reply:
> maybe they are using some awesome css-in-js library for it or something, so let's just go with it

We _are_ using the awesome https://theme-ui.com/ ((on top of [Emotion](https://emotion.sh/docs/introduction)) which e.g. grabs `headerHeight` from our theme's `sizes` object for us), but it doesn't magically work irt `env` ;-).

> I'll test with height removed right now!

Thank you so much! 🤗

Reply:
yup, on real devices it works as expected with height removed 😅
iPhone X:
![IMG_0114](https://user-images.githubusercontent.com/23037261/67215007-09fa2f80-f421-11e9-9400-b0d06c6c02ea.PNG)
iPhone 8:
![IMG_0930 2](https://user-images.githubusercontent.com/23037261/67215008-09fa2f80-f421-11e9-8cd9-0b24007b53d5.PNG)

Reply:
Sweet deal! Let's get this in!